modules/quickstart/pages/network-quickstart.adoc

```diff
@@ -159,13 +330,17 @@ npm install
 ----
 +
 For more information about this step, see link:../developers-guide/webpack-config{outfilesuffix}#troubleshoot-node[Ensuring node is available in a project].
+
 . Register, build, and deploy your first application by running the following command:
 +
 [source,bash]
 ----
-dfx deploy --network=ic
+dfx deploy --network ic
 ----
 +
+The `+--network+` option specifies the network alias or URL for deploying the application.
+This option is required to install on the {IC} running remotely.
```

Comment:
NIT: This option is required to remotely run your application on the IC.

Reply:
actually it really does mean install/deploy on the IC running remotely (instead of locally). Remotely is meant as the modifier for the IC rather than as a modifier for where your app runs. Ultimately, it is kind of the same thing, but that's the reasoning behind why it is the way it is.